packages/nhsuk-frontend/src/nhsuk/components/header/template.njk

```diff
@@ -52,7 +52,15 @@
   <div class="nhsuk-header__container nhsuk-width-container {%- if params.containerClasses %} {{ params.containerClasses }}{% endif %}">
     <div class="nhsuk-header__service">
     {% if logoHref %}
-      <a class="nhsuk-header__service-logo" href="{{ logoHref }}">
+      {% switch true %}
```

Comment:
It certainly works, but I'm not fully sure about this reverse switch syntax 😶

Regarding switch statements in general, I'm not sure if we should have used it based on [some interesting discussions](https://github.com/ministryofjustice/hmpps-sentence-plan-ui/pull/382#discussion_r1946870762). Similarly our [footer `switch` statement](https://github.com/nhsuk/nhsuk-frontend/blob/v10.0.0/packages/nhsuk-frontend/src/nhsuk/components/footer/template.njk#L5-L15) needed [rewriting for Jinja](https://github.com/NHSDigital/nhsuk-frontend-jinja/blob/v0.4.1/nhsuk_frontend_jinja/templates/nhsuk/components/footer/template.jinja#L5-L14)

What do you think?

Reply:
Updated.